app/scripts/lib/stream-logger.js

```diff
@@ -0,0 +1,61 @@
+import { createProjectLogger, createModuleLogger } from '@metamask/utils';
```

Comment:
Is it possible to use TypeScript for this file?

Reply:
fixed 97fbbce74